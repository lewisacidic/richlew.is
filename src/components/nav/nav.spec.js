import React from 'react'
import { render } from 'react-testing-library'

import Nav from './nav'

describe('Nav', () => {
  it('works', () => {
    const { container } = render(<Nav />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
