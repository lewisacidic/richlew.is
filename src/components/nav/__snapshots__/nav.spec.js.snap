// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav works 1`] = `
.c0 {
  list-style-type: none;
  text-align: center;
  margin-bottom: 0;
}

<ul
  class="c0"
/>
`;
